python/datashaper/tests/verbs/snapshot_test.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+
+from datashaper.engine.verbs import VerbInput, VerbManager
+from datashaper.table_store.types import TableContainer, VerbResult
+
+
+def make_verb_input(data: list, columns: list[str]):
+    pd_table = pd.DataFrame(data=data, columns=columns)
+    table_container = TableContainer(pd_table)
+    return VerbInput(table_container)
+
+
+def test_snapshot_csv():
+    verb_input = make_verb_input([[1], [2], [3], [4], [5]], ["id"])
+    window = VerbManager.get().get_verb("snapshot").func
+    output: VerbResult = window(input=verb_input, name="test-file", file_type="csv")
+    output: TableContainer = output.output
+
+    assert 5 == 5
```

Comment:
do you need to put actual asserts here for the output?

Reply:
Sorry I forgot to push the latest version on this file without these asserts